content/blog/r4ds-2e/index.Rmd

```diff
@@ -0,0 +1,62 @@
+---
+output: hugodown::hugo_document
+
+slug: r4ds-2e
+title: R for Data Science, 2nd edition
+date: 2023-07-11
+author: Mine Çetinkaya-Rundel
+description: >
+    The second edition of R for Data Science is out,
+    and it's a major reworking of the first edition.
+photo:
```

Comment:
This needs an update. Would be nice to use a photo of the actual book?

Reply:
Yup, I've made images with actual photo of the book, neglected to update the photo URL. Should I link to https://www.oreilly.com/library/view/r-for-data/9781492097396/ with "O'Reilly" as the text or https://r4ds.hadley.nz/ with "R for Data Science" as the text, any preference?

Reply:
I meant like take a photo of it sitting on your desk 😄